Update _index.md (#9135)
DataDog/documentation: content/en/agent/basic_agent_usage/_index.md

| [Windows 64-bit][10]                            | Windows 7+                                        |
| [Windows Azure Stack HCI OS][10]                | All Versions                                      |

> **Note**: 

Reviewer: ```suggestion
**Notes**: 
```

- [Source][11] install may work on operating systems not listed here and is supported on a best effort basis.

> - Datadog Agent v7+ supports Windows Server 2008 R2+ with the most recent Windows updates installed. There is also a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.

Reviewer 2: Unless I'm missing something this note only applies to Windows Server 2008 R2 specifically, not later Windows versions:

```suggestion
- Datadog Agent v7+ supports Windows Server 2008 R2 with the most recent Windows updates installed. There is also a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2.
```

> - Windows Server 2008 R2+ is supported, but must have the most recent updates installed in order to run versions 7+ of the Datadog Agent. There is also [a known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.

Reviewer 3: I think we would need the update to run the latest version of 6 too, no ?

Author: The issue referenced is specific to Python3, some Windows Updates have to be installed for it to work. Agent 6 is fine as long as they are using Python2.7.

> - Windows Server 2008 R2+ is supported, but must have the most recent updates installed in order to run versions 7+ of the Datadog Agent. There is also [a known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.

Reviewer: ```suggestion
- Windows Server 2008 R2+ is only supported on the most recent Datadog Agents v6 and v7. However, there is a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.
```

Author: @ruthnaebeck I want to clarify that by "must have the most recent updates installed", it's referring to Windows update and not Agent update. How would you suggest we make this clear?

Reviewer: @tanjiehui - Updated my feedback

- [Source][11] install may work on operating systems not listed here and is supported on a best effort basis.

> - Windows Server 2008 R2+ is supported, but there is [a known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.

Reviewer: ```suggestion
- Windows Server 2008 R2+ is supported, but there is a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.
```

> - Windows Server 2008 R2+ is supported, but there is a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.

Reviewer 2: Ditto:

```suggestion
- Windows Server 2008 R2 is supported, but there is a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2.
```
